### ISSUE TITLE
Add ability to save figure as multiple filetypes simultaneously

### ISSUE THREAD

**Issue by reillyowencooper** [Arcadia-Science/arcadiathemeR#35]
I often want to save a .pdf and .png file for each figure I make, which results in a lot of duplicate lines of code where the only thing I've changed is the extension. It would be great to have an option in `save_arcadia_plot` that allows you to save both filetypes (or more, if people use other extensions) at the same time. Something like:

```
save_arcadia_plot <- function(filepath, plot, file_extensions=c("pdf"), panel_size = c("full_wide", "float_wide", "half_square", "full_square", "float_square")) {
  [panel size specifications]
  
  # Use ggsave to save the plot
  for (ext in file_extensions) {
    if (ext == "pdf") {
      device <- "pdf"
    } else if (ext == "png") {
      device <- "png"
    } else {
      device <- ext # I'm not sure if others need to be specified individually
    }
    ggplot2::ggsave(
      filename = filepath,
      plot = plot,
      device = device,
      dpi = 72,
      width = width,
      height = height,
      units = "px",
      bg = 'transparent'
    )
  }
}
```

**Comment by elizabethmcd** [comment]
Addressed in #34 - please take a look at the PR and see if the way I have it set up makes sense, and I can close after you/others approve

**Comment by elizabethmcd** [comment]
Addressed and implemented in #36
